README.rst

```diff
@@ -8,6 +8,41 @@ A library to retrieve coordinates from an google account that has been shared lo
 * Documentation: https://locationsharinglib.readthedocs.org/en/latest
 
 
+
+Getting Started
+====================
```

Comment:
This would need to be covering only the text above as per markdown.

Reply:
I removed this